test_conformance/extensions/cl_khr_command_buffer/cl_khr_command_buffer_mutable_dispatch/mutable_command_out_of_order.cpp

```diff
@@ -205,19 +195,33 @@ struct OutOfOrderTest : public BasicMutableCommandBufferTest
     //--------------------------------------------------------------------------
     cl_int RunSingle()
     {
-        cl_int error = RecordCommandBuffer();
+        cl_int error;
+
+        if (!user_single_event)
+        {
+            user_single_event = clCreateUserEvent(context, &error);
+            test_error(error, "clCreateUserEvent failed");
+        }
```

Comment:
This shouldn't be a user created event (this is simultaneous test specific), we can remove this code as it will be initalized as the return parameter to `clEnqueueCommandBufferKHR`.  Would also remove the `user` part of the variable name.